lib/features/email/presentation/controller/single_email_controller.dart

```diff
@@ -1159,13 +1159,21 @@ class SingleEmailController extends BaseController with AppLoaderMixin {
   }
 
   Future<void> openMailToLink(Uri? uri) async {
+    if (uri == null) return;
+    
+    final navigationRouter = RouteUtils.generateNavigationRouterFromMailtoLink(uri.toString());
     log('SingleEmailController::openMailToLink(): ${uri.toString()}');
-    String address = uri?.path ?? '';
-    log('SingleEmailController::openMailToLink(): address: $address');
-    if (address.isNotEmpty) {
-      final emailAddress = EmailAddress(null, address);
-      mailboxDashBoardController.goToComposer(ComposerArguments.fromEmailAddress(emailAddress));
-    }
+    if (navigationRouter.listEmailAddress?.isEmpty ?? true) return;
```

Comment:
Should allow composer to open when one of the other params (listEmailAddress, cc, bcc, subject, body) is not empty

Reply:
Done